examples/qr-generator/README.md

```diff
@@ -1,5 +1,14 @@
 # QR code generator
 
-Toolpad application that defines a backend function to encode some text into a QR code.
+<p class="description">A basic Toolpad application that defines a backend function to encode some text/URL into a QR code.</p>
```

Comment:
Can we add some tutorial/descriptive instructions in this? I am contrasting with this [benchmark](https://interval.com/examples/qr-codes)